assets/python/ml/pyml:model:adaboosted_trees_regression:sklearn.j2.py

```diff
@@ -0,0 +1,95 @@
+# ----------------------------------------------------------------- #
+#                                                                   #
+#   Workflow unit for adaboosted trees regression with              #
+#   Scikit-Learn. Parameters for the estimator and ensemble         #
+#   are derived from sklearn defaults.                              #
+#                                                                   #
+#   When then workflow is in Training mode, the model is trained    #
+#   and then it is saved, along with the RMSE and some              #
+#   predictions made using the training data (e.g. for use in a     #
+#   parity plot or calculation of other error metrics).             #
+#                                                                   #
+#   When the workflow is run in Predict mode, the model   is        #
+#   loaded, predictions are made, they are un-transformed using     #
+#   the trained scaler from the training run, and they are          #
+#   written to a filed named "predictions.csv"                      #
+# ----------------------------------------------------------------- #
+
+import sklearn.ensemble
+import sklearn.tree
+import sklearn.metrics
+import numpy as np
+import settings
+
+with settings.context as context:
+    # Train
+    if settings.is_workflow_running_to_train:
+        # Restore the data
+        train_target = context.load("train_target")
+        train_descriptors = context.load("train_descriptors")
+        test_target = context.load("test_target")
+        test_descriptors = context.load("test_descriptors")
+
+        # Flatten the targets
+        train_target = train_target.flatten()
+        test_target = test_target.flatten()
+
+        # Initialize the base regressor
+        base_estimator = sklearn.tree.DecisionTreeRegressor(criterion="mse",
```

Comment:
I assume that the parameters would be the main source of changes - should we include them above right after imports, like:

decision_tree_regressor_parameters = {
    "splitter": "best",
    "max_depth": None,
    "min_samples_split": 2,
    "min_samples_leaf": 1,
    "min_weight_fraction_leaf": 0.0,
    "max_features": None,
    "max_leaf_nodes": None,
    "min_impurity_decrease": 0.0,
    "ccp_alpha": 0.0,
}

and similarly for AdaBoostRegressor. And then pass as kwargs?

Reply:
Keeping these the way they are per conversation